docs/data/core-edit-post.md

```diff
@@ -0,0 +1,271 @@
+# **core/edit-post**: The Editor's UI Data
+
+## Selectors 
+
+### getEditorMode
+
+Returns the current editing mode.
+
+*Parameters*
+
+ * state: Global application state.
+
+
+### isEditorSidebarOpened
+
+Returns true if the editor sidebar is opened.
```

Comment:
It would be nice to wrap "true" in backticks so it was highlighted: eg: <code>\`true\`</code>